blockchain/chain/src/store/chain_store.rs

```diff
@@ -385,7 +397,7 @@ where
     }
 
     /// Constructs and returns a full tipset if messages from storage exists - non self version
-    pub fn fill_tipset(&self, ts: Arc<Tipset>) -> Result<FullTipset, Arc<Tipset>>
+    pub fn fill_tipset(&self, ts: &Tipset) -> Option<FullTipset>
```

Comment:
ah oops good call this is no longer needed, was an artifact from when Tipset was passed in and was not in wrapped in an Arc and this was a benefit